src/pages/PlayPage.js

```diff
@@ -25,10 +24,10 @@ export default function PlayPage({
   useShortcut(['c'], onAnswer)
   useShortcut(['3'], onAnswer)
   useShortcut(['s'], onPlay)
-
+  const isCounter = Boolean(counter > 0)
```

Comment:
interesting!

Reply:
Yea run into a ["React renders 0 in conditional rendering issue" ](https://www.designcise.com/web/tutorial/why-is-react-showing-0-when-conditionally-rendering-a-component)